CHANGELOG.md

```diff
@@ -1,7 +1,39 @@
+* Fix nested program parsing ([86b8784](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/86b87842777b2893c060b9ba997e0b971e97d2a6)), closes [#730](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/730) [#596](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/596)
+* Fix REPLACE clause when replacing or replaced text contains "BY" [#900](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/900) ([7d88dc7](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/7d88dc70c5a32f4116fc7dac302722c21ed93e44))
+* Fix syntax error on READY/RESET TRACE ([5871b0b](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/5871b0bc161e2c1d8a834c6ee4f07784fe396977))
+* Fix integration with Explorer for Endevor [#922](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/922) ([398dc49](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/398dc49447219603c4bc5119e972ab8265313f24))
+* Fix the bug with two COPY REPLACING statements in a row ([035b661](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/035b661fb0d0f0dcad41a50c4837cf99a5380b51))
+* Fix color scheme ([3a383be](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/3a383be7f96ece655637b8c212be8a30069f9790))
+* Fix performance issues
+
+### Features
+* Add integration with COBOL Control Flow extension [#801](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/801) ([1d9a2e6](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/1d9a2e66f7f0c6af9b28f2fbf14d1371dd21be8f))
+* Add global variable support ([91e9b59](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/91e9b5957ebc59b51507d6473577b81293e011d9)), closes [#730](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/730)
+* Add IDMS DML starting keyword descriptions [#704](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/704) ([45baa15](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/45baa1527ed578662602368d012bee5547e83605))
+* Expand support for IDMS DML commands [#704](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/704) ([e908c8f](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/e908c8f1466b0497fd8b238cdb193ad2d5947577)) ([286c42e](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/286c42ef50eb91da4aa826aba07a93158b1026df))  ([ca0cf91](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/ca0cf9120128f29892a226d2e587c0a098e31e74)) ([91646a4](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/91646a441ce94104c29c4a64820ef021f3d58072)) ([f55cdfc](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/f55cdfc059b4834c9e7a5c163e00a09947341827)) ([05020f9](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/05020f9d0b2558024fb8ce07885b6f5902aaf4f8))
+* add support for Datacom SQLDA and SQLCA implicit fields [#885](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/885) ([e6944d3](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/e6944d3340e65fa9f93a154c8f86ff92854c54e8))
```

Comment:
Don't forgot about https://github.com/eclipse/che-che4z-lsp-for-cobol/pull/937

Reply:
Ok, I will add the commit mention here once the PR is merged.